src/sentry/snuba/sessions.py

```diff
@@ -266,6 +264,23 @@ def get_release_health_data_overview(
             ),
             "has_health_data": True,
         }
+        if len(x["duration_quantiles"]) == 2:
+            rp.update(
+                {
+                    "duration_p50": _convert_duration(x["duration_quantiles"][0]),
+                    "duration_p90": _convert_duration(x["duration_quantiles"][1]),
+                }
+            )
+        else:
+            rp.update(
+                {
+                    "duration_p50": _convert_duration(x["duration_quantiles"][0]),
+                    "duration_p75": _convert_duration(x["duration_quantiles"][1]),
```

Comment:
Not quite sure if I should output these new quantiles right away, or rather in a separate PR?
On the other hand, I think only `p50` is actually used by the frontend, right?

Reply:
Yeah, only p50 is used